packages/design-system/src/components/Dropdown/Dropdown.stories.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+import Dropdown from './Dropdown';
+import Select from './Select';
+
+export default {
+  title: 'Components/Dropdown',
+  component: Dropdown,
+  argTypes: {
+    children: { control: false },
+    options: { control: false },
+  },
+  parameters: {
```

Comment:
When I originally setup storybook, I didn't realize the backgrounds would be used so frequently. Since this code is repeated in a number of component story files, this config can be elevated to be global parameters.

for example in `preview.js`, this chunk of code would be moved to the parameters property
```
backgrounds: {
    default: 'light',
    values: [
      {
        name: 'light',
        value: '#fff',
      },
      {
        name: 'Hcgov dark',
        value: '#112e51',
      },
      {
        name: 'Mgov dark',
        value: '#146a5d',
      },
    ],
  },
```

Reply:
Not sure if you want to update that here, or if a separate PR is better, but doing this will prevent the copy/paste for every component that has an inverse example

Reply:
Yeah I agree, it's a bad pattern we should squash early.. I can make the changes here for all components using it, or open a new pr with changes for everything.. I thing a new pr would be cleaner in case we ever had to back it out?

Reply:
I'm going to write a quick ticket for next sprint and do this for all modules using this pattern next sprint, will ping Krithi with it.